antarest/study/storage/rawstudy/io/reader/ini_reader.py

```diff
@@ -77,6 +77,53 @@ def read(self, path: Path) -> JSON:
         }
 
 
+class XpansionSettingsIniReader(IReader):
+    """
+    Standard .ini file reader. Use for general purpose.
+    """
+
+    @staticmethod
+    def _parse_inf(value: str) -> Optional[str]:
+        try:
+            return "inf" if float(value) == float("inf") else None
+        except ValueError:
+            return None
+
+    @staticmethod
+    def parse_value(value: str) -> ELEMENT:
+        parsed: Union[
+            str, int, float, bool, None
+        ] = XpansionSettingsIniReader._parse_inf(value)
+        parsed = parsed if parsed is not None else IniReader._parse_bool(value)
+        parsed = parsed if parsed is not None else IniReader._parse_int(value)
+        parsed = (
+            parsed if parsed is not None else IniReader._parse_float(value)
+        )
+        return parsed if parsed is not None else value
+
+    @staticmethod
+    def _parse_json(json: configparser.SectionProxy) -> JSON:
+        return {
+            key: XpansionSettingsIniReader.parse_value(value)
+            for key, value in json.items()
+        }
+
+    def read(self, path: Path) -> JSON:
+        config = IniConfigParser()
```

Comment:
why use IniConfigParser ?
the `_parse_json` can handle raw python dict just fine